skips over empty ROIs

djkapner opened the pull request:
When running 126 ophys experiments, 5 failed with an empty coo matrix. This PR implements a skipping-over of these ROIs.

### Validation
These 5 runs failed with the current master image:
```
[
{
  "MotionCorrectedImageStack": "/allen/programs/braintv/production/visualbehavior/prod2/specimen_887264964/ophys_session_960593969/ophys_experiment_960960480/processed//motion_corrected_video.h5",
  "OphysMotionXyOffsetData": "/allen/programs/braintv/production/visualbehavior/prod2/specimen_887264964/ophys_session_960593969/ophys_experiment_960960480/processed//960960480_rigid_motion_transform.csv",
  "id": 960960480
}
{
  "MotionCorrectedImageStack": "/allen/programs/braintv/production/visualbehavior/prod2/specimen_887264964/ophys_session_957189583/ophys_experiment_957652800/processed//motion_corrected_video.h5",
  "OphysMotionXyOffsetData": "/allen/programs/braintv/production/visualbehavior/prod2/specimen_887264964/ophys_session_957189583/ophys_experiment_957652800/processed//957652800_rigid_motion_transform.csv",
  "id": 957652800
}
{
  "MotionCorrectedImageStack": "/allen/programs/braintv/production/visualbehavior/prod3/specimen_910267443/ophys_session_995560574/ophys_experiment_995622557/processed//motion_corrected_video.h5",
  "OphysMotionXyOffsetData": "/allen/programs/braintv/production/visualbehavior/prod3/specimen_910267443/ophys_session_995560574/ophys_experiment_995622557/processed//995622557_rigid_motion_transform.csv",
  "id": 995622557
}
{
  "MotionCorrectedImageStack": "/allen/programs/braintv/production/visualbehavior/prod5/specimen_989489224/ophys_session_1012635881/ophys_experiment_1012746738/processed//motion_corrected_video.h5",
  "OphysMotionXyOffsetData": "/allen/programs/braintv/production/visualbehavior/prod5/specimen_989489224/ophys_session_1012635881/ophys_experiment_1012746738/processed//1012746738_rigid_motion_transform.csv",
  "id": 1012746738
}
{
  "MotionCorrectedImageStack": "/allen/programs/braintv/production/visualbehavior/prod3/specimen_916452082/ophys_session_990144181/ophys_experiment_990381322/processed//motion_corrected_video.h5",
  "OphysMotionXyOffsetData": "/allen/programs/braintv/production/visualbehavior/prod3/specimen_916452082/ophys_session_990144181/ophys_experiment_990381322/processed//990381322_rigid_motion_transform.csv",
  "id": 990381322
}
]
```
with this change, these no longer fail and go through extract traces.

kschelonka commented:
Thanks Dan. Can you please add the None test cases to the appropriate functions as well? e.g. crop_roi_mask

djkapner commented:
> Thanks Dan. Can you please add the None test cases to the appropriate functions as well? e.g. crop_roi_mask

Originally, I hadn't needed to touch that function, but it was a little wonky, so I modified it to also return `None` and not raise a `ValueError`. Modified the test.
